JSDemos/Demos/VectorMap/DynamicViewport/description.md

```diff
@@ -1 +1,5 @@
-This demo illustrates how to&nbsp;change the map&rsquo;s viewport from code. Choose a&nbsp;continent using the drop-down menu under the map. In&nbsp;response to&nbsp;that, the map changes the visible area so&nbsp;that the chosen continent is&nbsp;displayed optimally. This is&nbsp;performed in&nbsp;code by&nbsp;calling the **viewport(viewportCoordinates)** method. After that, callback functions assigned to&nbsp;the **centerChanged** and **zoomFactorChanged** properties are triggered. These functions place the current values of&nbsp;the map&rsquo;s center and zoom factor in&nbsp;the text boxes under the map.
\ No newline at end of file
+This demo illustrates how to change the map viewport. Use the drop-down menu under the map to choose a continent. In response to that, the map changes the visible area so that the chosen continent is displayed optimally. 
+
+To implement this functionality, call the [viewport(viewportCoordinates)](/Documentation/ApiReference/UI_Components/dxVectorMap/Methods/#viewportviewportCoordinates) method every time the drop-down box value changes. You can also implement the [onCenterChanged](/Documentation/ApiReference/UI_Components/dxVectorMap/Configuration/#onCenterChanged) and [onZoomFactorChanged](/Documentation/ApiReference/UI_Components/dxVectorMap/Configuration/#onZoomFactorChanged) functions. These functions place the current values of the map’s center and zoom factor in the text boxes under the map.
+
+To specify the [control bar](/Documentation/ApiReference/UI_Components/dxVectorMap/Configuration/controlBar/) visibility, enable or disable [panVisible](/Documentation/ApiReference/UI_Components/dxVectorMap/Configuration/controlBar/#panVisible) and [zoomVisible](/Documentation/ApiReference/UI_Components/dxVectorMap/Configuration/controlBar/#zoomVisible) properties. You can toggle switches under the map to see how it affects the control bar. 
```

Comment:
To specify the visibility. Yes, in theory, you can say like that. However, we seek to be as close to real-life tasks as possible. To show or hide the control bar...

If you use property names along with the word "property" or "properties", you surely need a definite article.

> to see how **it** affects the control bar.

"It" what? While it's "obvious", we still should seek to be more specific at everything.

```suggestion
To show or hide the [control bar](/Documentation/ApiReference/UI_Components/dxVectorMap/Configuration/controlBar/), enable or disable the [panVisible](/Documentation/ApiReference/UI_Components/dxVectorMap/Configuration/controlBar/#panVisible) and [zoomVisible](/Documentation/ApiReference/UI_Components/dxVectorMap/Configuration/controlBar/#zoomVisible) properties. You can toggle switches under the map to see how these settings affect the control bar. 
```